MekHQ/src/mekhq/campaign/parts/AeroSensor.java

```diff
@@ -51,7 +51,7 @@ public AeroSensor(int tonnage, boolean lc, Campaign c) {
         super(tonnage, c);
         this.name = "Aerospace Sensors";
         this.largeCraft = lc;
```

Comment:
I know it'll likely add a bunch of extra changes, but can we neuter these abbreviated and single-letter variable names?

Reply:
Went through and removed single-letter/abbreviated names in AeroSensor and MissingAeroSensor constructors

Reply:
I know it's a pain in the bum, but it is appreciated. Single letter and abbreviation variable names easily fall into the track of being completely obvious to the code author, but ten years later are incomprehensible. So, as a gift for Future Us, we're replacing them as they're found.